TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+        // Needed to compile multiline strings in Cesium
+        sourcePrefix: ''
+    },
+    amd: {
+        // Enable webpack-friendly use of require in Cesium
+        toUrlUndefined: true
+    },
+    resolve: {
+        alias: {
+            cesium: path.resolve(__dirname, cesiumSource)
+        },
+        mainFiles: ['module', 'main', 'Cesium']
+    },
+    module: {
+        rules: [{
+            test: /\.css$/,
+            use: [ 'style-loader', 'css-loader' ]
+        }, {
+            test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
+            use: [ 'url-loader' ]
+        }]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html'
+        })
+    ],
+    mode: 'development',
+};
+```
+
+## Manage CesiumJS static files
+1. Now, we must make sure the static CesiumJS asset, widget, and web worker files are served and loaded correctly. Use `copy-webpack-plugin` to copy static files to the `dist` directory as part of the build process. To do this run the command `npm install --save-dev copy-webpack-plugin` and updated the plugins array in `webpack.config.js`. `webpack.config.js` should now look like
+```
+// The path to the CesiumJS source code
+const cesiumSource = 'node_modules/cesium/Source';
+const cesiumWorkers = '../Build/Cesium/Workers';
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    context: __dirname,
+    entry: {
+        app: './src/index.js'
+    },
+    output: {
+        filename: 'app.js',
+        path: path.resolve(__dirname, 'dist'),
+        sourcePrefix: ''
+    },
+    amd: {
+        // Enable webpack-friendly use of require in Cesium
+        toUrlUndefined: true
+    },
+    resolve: {
+        alias: {
+            cesium: path.resolve(__dirname, cesiumSource)
+        },
+        mainFiles: ['module', 'main', 'Cesium']
+    },
+    module: {
+        rules: [{
+            test: /\.css$/,
+            use: [ 'style-loader', 'css-loader' ]
+        }, {
+            test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
+            use: [ 'url-loader' ]
+        }]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html'
+        }),
+        // Copy Cesium Assets, Widgets, and Workers to a static directory
+        new CopyWebpackPlugin({ 
+            patterns: [
+                { from: path.join(cesiumSource, cesiumWorkers), to: 'Workers' },
+                { from: path.join(cesiumSource, 'Assets'), to: 'Assets' },
+                { from: path.join(cesiumSource, 'Widgets'), to: 'Widgets' }
+            ]
+        }),
+        new webpack.DefinePlugin({
+            // Define relative base path in cesium for loading assets
+            CESIUM_BASE_URL: JSON.stringify('')
+        })
+    ],
+    mode: 'development',
+};
+```
+
+## Incorporate CesiumJS
```

Comment:
This is very vague

Reply:
I added a commit that changes this to "Incorporate CesiumJS Into Your Application"

Reply:
Incorporate isn't usually used in a coding sense. Are you sure its the right term here?